Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:8.10
+
+# Manually install missing shared libs for Chromium.
+RUN apt-get update && \
+apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
```

Comment:
libcairo2? +cc @marwanad.

Reply:
`Cairo is a multi-platform library providing anti-aliased vector-based rendering` - straight outta Egypt